doc/papers/joss/paper.bib

```diff
@@ -0,0 +1,441 @@
+  year = "2021"
+}
+
+@misc{OCCAwebsite,
+  title = "{OCCA} development site",
+  url = "http://github.com/libocca/occa",
+  year = "2021"
+}
+
+@article{Orszag:1980,
+  title = "Spectral methods for problems in complex geometries",
+  journal = "Journal of Computational Physics",
+  volume = "37",
+  pages = "70 - 92",
+  year = "1980",
+  doi = "10.1016/0021-9991(80)90005-4",
+  author = "Steven A Orszag"
+}
+
+@TechReport{PETScUserManual,
+  author = {Satish Balay and Shrirang Abhyankar and Mark~F. Adams and Jed Brown and Peter Brune
+            and Kris Buschelman and Lisandro Dalcin and Alp Dener and Victor Eijkhout and William~D. Gropp
+            and Dmitry Karpeyev and Dinesh Kaushik and Matthew~G. Knepley and Dave~A. May and Lois Curfman McInnes
+            and Richard Tran Mills and Todd Munson and Karl Rupp and Patrick Sanan
+            and Barry~F. Smith and Stefano Zampini and Hong Zhang and Hong Zhang},
+  title  = {{PETS}c Users Manual},
+  institution = {Argonne National Laboratory},
+  year   = 2021,
+  number = {ANL-95/11 - Revision 3.15}
+}
+
+@misc{Python-lang,
+  title = "Python",
+  url = "https://www.python.org/",
+  year = "2021",
+}
+
+@misc{Rust-lang,
+  title = "Rust",
+  url = "https://www.rust-lang.org/",
+  year = "2021",
+}
+
+@misc{python-cffi,
+  title = "C Foreign Function Interface for {Python}",
+  url = "https://cffi.readthedocs.io",
+  year = "2021",
+}
+
+@article{straka1993numerical,
+  title={Numerical solutions of a non-linear density current: A benchmark solution and comparisons},
+  author={Straka, Jerry M and Wilhelmson, Robert B and Wicker, Louis J and Anderson, John R and Droegemeier, Kelvin K},
+  journal={International Journal for Numerical Methods in Fluids},
+  volume={17},
+  number={1},
+  pages={1--22},
+  year={1993},
+  publisher={Wiley Online Library},
+  doi={10.1002/fld.1650170103}
+}
+
+@book{deville2002highorder,
+  title={High-order methods for incompressible fluid flow},
+  author={Deville, Michel O and Fischer, Paul F and Mund, Ernest H},
+  year={2002},
+  publisher={Cambridge University Press},
+  isbn={0-521-45309-7}
+}
+
+@article{gropp2014using,
+  title={Using {MPI}: Portable Parallel Programming with the Message-Passing Interface},
+  author={Gropp, William and Lusk, Ewing and Skjellum, Anthony},
+  year={2014},
+  publisher={The MIT Press},
+  isbn={9780262527392}
+}
+
+@article{libMeshPaper,
+  author = {Benjamin S. Kirk and John W. Peterson and Roy H. Stogner and Graham F. Carey},
+  title = {{\texttt{libMesh}: A C++ Library for Parallel Adaptive Mesh Refinement/Coarsening Simulations}},
+  journal = {Engineering with Computers},
+  volume = 22,
+  number = {3--4},
+  pages = {237--254},
+  year = 2006,
+  doi = {10.1007/s00366-006-0049-3}
+}
+@book{LoggMardalWells2012,
+  title  = {Automated solution of differential equations by the finite element method: The {FEniCS} book},
+  title = {Automated Solution of Differential Equations by the Finite Element Method},
```

Comment:
Thanks @jedbrown for these edits and updates. Which one of these two titles should be used for the Fenics book?

Reply:
Thanks for the catch. The cover and [publisher](https://www.springer.com/gp/book/9783642230981#aboutBook) includes `The FEniCS book` subtitle, but ["how to cite" does not](https://fenicsproject.org/book/). I'd prefer to include it (i.e., keep the top line above) because it completely clarifies to any reader who has heard of the book, while the other title is somewhat generic to other publications by the authors.

Reply:
```suggestion
```